jupyterlab/staging/package.json

```diff
@@ -211,7 +211,7 @@
     "yarn-deduplicate": "^5.0.0"
   },
   "engines": {
-    "node": ">=12.0.0"
```

Comment:
Normally the files under `jupyterlab/staging` don't need to be updated manually (done during the release process). So we can revert that change.